extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -24,59 +25,73 @@ Draft
 
 Written against the 3D Tiles 1.0 specification.
 
+<!-- omit in toc -->
+## Optional vs. Required
+
+This extension is required, meaning that it must be listed in
+both the `extensionsUsed` and `extensionsRequired` arrays of the
+tileset JSON.
+
 <!-- omit in toc -->
 ## Contents
 
 - [Overview](#overview)
```

Comment:
The TOC is still a bit long and might seem daunting. Some suggestions:

* Subdivision scheme and Subdivision rules can be collapsed into one section
* These sections don't need sub-bullets
  * Extension JSON schema reference
  * Subtree JSON schema reference
  * Appendix A: Availability Indexing